src/library/FOSSBilling/UpdatePatcher.php

```diff
@@ -156,7 +159,7 @@ private function executeSql($sql): void
         $statement = $this->di['pdo']->prepare($sql);
         try {
             $statement->execute();
-        } catch (\Box_Exception $e) {
+        } catch (\FOSSBilling\Exception $e) {
```

Comment:
Just one... or two left. 😉

Reply:
![sddefault](https://github.com/FOSSBilling/FOSSBilling/assets/17304943/6882fcfc-c124-42dc-ba16-4f1d3fbd4cfc)

Reply:
Actually.. I think this one is just outright an error. This should be catching any exception rather than just the FOSSBilling specific one. I will correct this in the code